sky/cli.py

```diff
@@ -208,15 +208,53 @@ def _interactive_node_cli_command(cli_func):
                               default=False,
                               required=False,
                               help='Skip confirmation prompt.')
+    idle_autostop = click.option('--idle-minutes-to-autostop',
```

Comment:
`--down` option is also needed to decide whether this is `autostop` or `autodown`